pkg/observability/k8s-events.go

```diff
@@ -81,8 +87,8 @@ func InitK8sEventRecorder(enabled bool, nodeName string, sqsMode bool, nodeMetad
 	}
 
 	annotations := make(map[string]string)
+	annotations["account-id"] = nodeMetadata.AccountId
 	if !sqsMode {
-		annotations["account-id"] = nodeMetadata.AccountId
 		annotations["availability-zone"] = nodeMetadata.AvailabilityZone
 		annotations["instance-id"] = nodeMetadata.InstanceID
 		annotations["instance-life-cycle"] = nodeMetadata.InstanceLifeCycle
```

Comment:
I agree with your decision to leave `instance-life-cycle` out of the annotations for SQS mode, since including it would require an additional AWS API call. If it turns out customers do want it later, we can discuss adding it in.

Reply:
We can also add a new node-labels-to-event-annotations feature. Similar to the extra annotations we already have, but in this case the user would define a set (or even all?) of `Node` object labels that NTH would add to the event annotations map. Then the user could label all spot/ondemand instances they way they see fit (or any other arbitrary label FWIW) and instruct NTH to add such label as event annotation. NTH calls the k8s API anyway to get the `Node` object, so this extra feature would come for free in terms of remote API calls.